ucx.spec

```
@@ -121,8 +122,8 @@ Provides header files and examples for developing with UCX.
 make %{?_smp_mflags} V=1
 
 %install
-make DESTDIR=%{buildroot} install
-rm -f %{buildroot}%{_libdir}/*.la
+make DESTDIR=%{buildroot} install prefix=%{_prefix} libdir=%{_libdir} datadir=%{_datadir}
```

These shouldn't be necessary.  IIUC the RPM builds were already working correctly so probably nothing in this file needs updating.

```suggestion
make DESTDIR=%{buildroot} install
```